### PR TITLE
[Team_Task0]: A Hello world pr

### PR DESCRIPTION
**Pull request by Vancool**
### What

- brief: To print "Hello World!" string. 
- author:  eva 

### Directory Specification

A hello world pr for testing. 

### What changes are included in this PR?

A hello world program & team info doc

### Are these changes tested?

Manual testing **PASS** 
image

### CODE REVIEW

File: stl/test/eva/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+#include <string>
+
+int main()
+{
+    std::cout << "Hello, World!" << std::endl;
+
```
Comment (Light-City) [inline stl/test/eva/src/main.cpp:7]: remove this line.

**Comment by Vancool** [comment]
This is an automatic reply, confirming that Lin  received your e-mail.Thank you!

**Comment by Light-City** [comment]
Write the code later to follow the google specification and pass it first。

**Comment by Light-City** [comment]
LGTM

**Pull request closed without merging**